doc/source/custom_directives.py

```diff
@@ -59,6 +59,7 @@ def update_context(app, pagename, templatename, context, doctree):
     "dask.distributed",
     "datasets",
     "datasets.iterable_dataset",
+    "datasets.load",
```

Comment:
What is this for?

Reply:
Render HF docs properly - a bit of a drive by change, but I couldn’t build the docs for this PR locally without it